keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone_presets.py

```diff
@@ -0,0 +1,478 @@
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "stackwise_kernel_sizes": [3, 3, 3, 3, 3, 3, 3],
+            "stackwise_num_repeats": [3, 5, 5, 7, 14, 18, 5],
+            "stackwise_input_filters": [24, 24, 48, 80, 160, 176, 304],
+            "stackwise_output_filters": [24, 48, 80, 160, 176, 304, 512],
+            "stackwise_expansion_ratios": [1, 4, 4, 4, 6, 6, 6],
+            "stackwise_squeeze_and_excite_ratios": [
+                0,
+                0,
+                0,
+                0.25,
+                0.25,
+                0.25,
+                0.25,
+            ],
+            "stackwise_strides": [1, 2, 2, 2, 1, 2, 1],
+            "stackwise_conv_types": [
+                "fused",
+                "fused",
+                "fused",
+                "unfused",
+                "unfused",
+                "unfused",
+                "unfused",
+            ],
+        },
+    },
+    "efficientnetv2-l": {
+        "metadata": {
+            "description": "The EfficientNet medium architecture.  In this "
+            "variant of the EfficientNet architecture, there are "
+            "7 convolutional blocks.  The primary distinction between the "
+            "EfficientNet large architecture and the EfficientNet medium lies "
+            "in the number of filters contained in each layer, and the number "
+            "of times each block is repeated."
+        },
+        "class_name": "keras_cv.models>EfficientNetV2Backbone",
+        "config": {
+            "width_coefficient": 1.0,
+            "depth_coefficient": 1.0,
+            "skip_connection_dropout": 0.2,
+            "depth_divisor": 8,
+            "min_depth": 8,
+            "activation": "swish",
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "stackwise_kernel_sizes": [3, 3, 3, 3, 3, 3, 3],
+            "stackwise_num_repeats": [4, 7, 7, 10, 19, 25, 7],
+            "stackwise_input_filters": [32, 32, 64, 96, 192, 224, 384],
+            "stackwise_output_filters": [32, 64, 96, 192, 224, 384, 640],
+            "stackwise_expansion_ratios": [1, 4, 4, 4, 6, 6, 6],
+            "stackwise_squeeze_and_excite_ratios": [
+                0,
+                0,
+                0,
+                0.25,
+                0.25,
+                0.25,
+                0.25,
+            ],
+            "stackwise_strides": [1, 2, 2, 2, 1, 2, 1],
+            "stackwise_conv_types": [
+                "fused",
+                "fused",
+                "fused",
+                "unfused",
+                "unfused",
+                "unfused",
+                "unfused",
+            ],
+        },
+    },
+    "efficientnetv2-b0": {
+        "metadata": {
+            "description": "The EfficientNet B0 architecture.  In this "
```

Comment:
Nit: Shorten to 
```
"EfficientNet B-style architecture with six convolutional blocks. This B-style model has `width_coefficient=1.0` and `depth_coefficient=1.0`."
``` 

This has to fit in a small table on keras.io. The scaling is clearly different for each B-style preset for there's no "all" that I can see.